recipes/pypairs/meta.yaml

```diff
@@ -1,36 +1,60 @@
 {% set name = "pypairs" %}
-{% set version = "2.0.6" %}
+{% set version = "3.0.9" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 39546c372405f22cefd2b00fdbf0bb2bd6abe779d88a35f82f4b746807446bcb
+  sha256: baf268424ebe0eb58339e41877531c182e4c6634f59e9c21b77e0f33ec2f634d
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
-  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py2k]
 
 requirements:
   host:
+    - anndata >=0.6.13
```

Comment:
I think most of these do not be in the host environment. I think only pip and python is needed.